SimplenoteUITests/SimplenoteUITestsSearch.swift

```diff
@@ -47,7 +47,7 @@ class SimplenoteUISmokeTestsSearch: XCTestCase {
         Sidebar.tagsDeleteAll()
         NoteList.openAllNotes()
 
-        allNotes.forEach { NoteList.createNoteThenLeaveEditor($0) }
+        allNotes.forEach { NoteList.createNoteThenLeaveEditor($0, viaPaste: true) }
```

Comment:
Not sure is `viaPaste` is the best name for this parameter

Reply:
It didn't stood out as odd to me, so feel free to keep it.

Other ideas:

```suggestion
        allNotes.forEach { NoteList.createNoteThenLeaveEditor($0, pastingContent: true) }
```

```suggestion
        allNotes.forEach { NoteList.createNoteThenLeaveEditor($0, usingPaste: true) }
```

Reply:
Went for `usingPaste`, thanks for the options!